ddtrace/ext/db.go

```diff
@@ -0,0 +1,12 @@
+package ext
+
+const (
+	// DBType indicates the type of Database.
+	DBType = "db.type"
+	// DBInstance indicates the instance name of Database.
+	DBInstance = "db.instance"
+	// DBUser indicates the user name of Database, e.g. "readonly_user" or "reporting_user".
+	DBUser = "db.user"
```

Comment:
Should we replace these values using the new constants throughout? We are using these keys in other integrations too.

Reply:
Good idea. Since this PR may take a while (depending on if the upstream changes can get merged), I'll break that into a separate PR.